DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -514,6 +514,20 @@
   <class name="edm::Wrapper< std::vector<edm::Ptr<pat::Jet> > >" />
   <class name="edm::Wrapper< std::vector< std::vector<edm::Ptr<pat::Jet> > > >" />
 
+  <class name="pat::PATTauDiscriminatorBase">
+    <field name="transientVector_" transient="true"/>
+  </class>  
+  <class name="pat::PATTauDiscriminator"  ClassVersion="3">
+   <version ClassVersion="3" checksum="2916098408"/>  
+  </class>  
```

Comment:
We did not invent this structure, we have the exact same situation (presumably since long) in https://github.com/cms-tau-pog/cmssw/blob/CMSSW_8_1_X_tau-pog/DataFormats/TauReco/src/classes_def_2.xml#L314 and following. 

So what am I meant to do now, add the member to the templated base class to the class derived from the templated base class or to both? If I do so should I do the same in DataFormats/TauReco/src/classes_def_2.xml?